src/main/resources/application.yml

```diff
@@ -21,10 +21,7 @@ spring:
     oauth2:
       resourceserver:
         jwt:
-          issuer-uri: ${oauth.endpoint.url}/issuer
```

Comment:
Not providing an issuer when running locally means that any locally generated JWT will be valid.  If an issuer is specified then it will be validated which means that only tokens issued on a matching port are valid which is a hurdle I wanted to remove.